starter-code/src/functions-and-arrays.js

```diff
@@ -1,16 +1,83 @@
 // Iteration #1: Find the maximum
 
+const maxOfTwoNumbers = function (num1, num2) {
+  // if num1 is bigger than num2, return num1
+  if (num1 > num2) {
+    return num1
+    // if num2 is bigger than num1, return num2
+  } else if (num2 > 1) {
+    return num2
+  } else {
+    // if they are equal, return error message
+    console.log('Your numbers are equal.')
+  }
+}
+
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+let findLongestWord = function (array) {
+  var longest = "";
+  // if there is only one item in the array, return that item
+  if (array.length === 1) {
+    return array[0];
+    // if the array is empty or undefined, return undefined
+  } else if (array.length < 1 || array == undefined) {
+    return undefined;
+  } else {
+    // loop through each item in the array and check for the length of each item. Each time the item's length is longer than the one stored in the 'longest' variable, overwrite it.
+    for (var i = 0; i < array.length; i++) {
+      if (longest.length < array[i].length) {
+        longest = array[i];
+      }
+    }
+    return longest;
+  }
+}
+
+findLongestWord(words);
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumArray(array) {
+  // !array.length is the same as writing array.length === 0
+  let sum = 0
+  if (!array.length) {
```

Comment:
Same as above, you don't need this 'if' statement. If your array is empty then the return sum will be returned with it's original value which is 0.